cluster/gce/gci/configure-helper.sh

```diff
@@ -85,11 +88,86 @@ function create-dirs {
   fi
 }
 
-# Formats the given device ($1) if needed and mounts it at given mount point
+# Gets the total number of $(1) and $(2) type disks specified
+# by the user in ${NODE_LOCAL_SSDS_EXT}
+function get-local-disk-num() {
+  local interface=$1
+  local format=$2
+
+  localdisknum=0
+  if [[ ! -z ${NODE_LOCAL_SSDS_EXT} ]]; then
+    IFS=";" read -r -a ssdgroups <<< ${NODE_LOCAL_SSDS_EXT}
+    for ssdgroup in "${ssdgroups[@]}"
+    do
+      IFS="," read -r -a ssdopts <<< ${ssdgroup}
+      local opnum=${ssdopts[0]}
+      local opinterface=${ssdopts[1]}
+      local opformat=${ssdopts[2]}
+
+      if [[ ${opformat,,} == ${format,,} && ${opinterface,,} == ${interface,,} ]]; then
+        localdisknum=$((localdisknum+opnum))
+      fi
+    done
+  fi
+}
+
+# Creates a symlink for a ($1) so that it may be used as block storage
+function safe-block-symlink(){
+  local device=$1
+  local symdir=$2
+  
+  mkdir -p ${symdir}
+
+  get-or-generate-uuid ${device}
+  local myuuid=${retuuid}
```

Comment:
nit: more quotes. (I'm not going to make this comment again. Just review for unquoted variables?)

Reply:
Yeah, sorry, this file is against style, but you're ok.

I would fix the for; dos, though.

Reply:
is quoting all variables recommended? I can at least quote all the variables in my changes.

Reply:
https://google.github.io/styleguide/shell.xml?showone=Quoting#Quoting